app/history_v2_test.go

```diff
@@ -106,6 +106,34 @@ func TestHistoryV2Handler(t *testing.T) {
 				g.Assert(messages[0].Blocked).Equal(false)
 
 			})
+
+			g.It("It should return 200 and only blocked messages if the user is authorized into the topic", func() {
```

Comment:
What if we send a wrong `isBlocked` flag? How the system will behave?

It could be good to have one more test case to it, it make sense?

Reply:
[Done, captain](https://github.com/topfreegames/mqtt-history/pull/20/commits/0ed990f5450f7cf036695a4193717ebaae3af4d6) 👍🏾